### DISCUSSION TITLE
TLDR Page Creation Wizard

### DISCUSSION THREAD

**Discussion by spageektti** [tldr-pages/tldr#12886] [Show and tell]
Hi. I started creating a wizard for creating a tldr page (still WIP).
It will guide you through the entire process of creating a tldr page and probably in the future also use GitHub API to send a PR.
I'm creating this in vanilla JS, CSS, and HTML (I have thought about React but haven't used it for a long time).

I am opening a discussion so you can say what you think about the idea because the website is unfinished ( and I won't finish it anytime soon).

A preview video:
[preview.webm](https://github.com/tldr-pages/tldr/assets/155078792/600c07a6-55f0-4995-ad07-c82b1f7acf0d)

**Comment by kant** [comment]
There is a typo: bellow -> below
